### PR TITLE
Enable for all teams

### PR DESCRIPTION
**Pull request by jSylvestre**

### CODE REVIEW

File: Keas.Mvc/ClientApp/components/Documents/DocumentsContainer.tsx [hunk 1]
```diff
@@ -22,29 +22,30 @@ export const DocumentsContainer = (props: IProps): JSX.Element => {
...
-    // TODO: once testing is done, turn on access for all teams
-    return (
-      ctx.team.slug === 'caes-cru' &&
-      PermissionsUtil.canViewDocuments(ctx.permissions)
-    );
-    // return PermissionsUtil.canViewDocuments(ctx.permissions);
-  }, [ctx]);
+  const canView = useMemo(
+    () => {
+      return PermissionsUtil.canViewDocuments(ctx.permissions);
```
Comment (jSylvestre) [inline Keas.Mvc/ClientApp/components/Documents/DocumentsContainer.tsx:27]: This is the only non formatting change

File: Keas.Mvc/ClientApp/components/Documents/DocumentsContainer.tsx [hunk 2]
```diff
@@ -82,37 +83,37 @@ export const DocumentsContainer = (props: IProps): JSX.Element => {
...
+            <i className="fas fa-plus fa-sm" aria-hidden="true" /> Add Document
           </Button>
         </div>
       </div>
-      <div className='card-content'>
+      <div className="card-content">
         <DocumentsList
           documents={documents}
           downloadUrl={`/api/${ctx.team.slug}/documents/get`}
-        ></DocumentsList>
```
Comment (jSylvestre) [inline Keas.Mvc/ClientApp/components/Documents/DocumentsContainer.tsx:108]: Prettier?

Comment (srkirkland) [reply Keas.Mvc/ClientApp/components/Documents/DocumentsContainer.tsx:108]: Yea but I wonder why my prettier is slightly different. I thought they would all use the ruleset that is in the repo.

**Comment by wongband** [comment]
@jSylvestre I solved the build failure. It was when the `DocumentsContainer` got added to the PeopleContainer and the jest tests didn't expect it. It was a quick fix after fixing an import and mocking the container.

**Comment by jSylvestre** [comment]
> @jSylvestre I solved the build failure. It was when the `DocumentsContainer` got added to the PeopleContainer and the jest tests didn't expect it. It was a quick fix after fixing an import and mocking the container.

@wongband  Thanks. I had tried the mock, but didn't make the other changes.

File: Keas.Mvc/ClientApp/components/Documents/DocumentsContainer.tsx [hunk 3]
```diff
@@ -15,36 +15,37 @@ interface IProps {
   person: IPerson;
 }
 
-export const DocumentsContainer = (props: IProps): JSX.Element => {
+const DocumentsContainer = (props: IProps): JSX.Element => {
```
Comment (jSylvestre) [inline Keas.Mvc/ClientApp/components/Documents/DocumentsContainer.tsx:18]: @srkirkland This is different from the other containers. but it looks like it works.

Comment (srkirkland) [reply Keas.Mvc/ClientApp/components/Documents/DocumentsContainer.tsx:18]: @jSylvestre yea, this is "new React" aka React with hooks.

**Pull request merged**